Feature/cds 13 record seg

chloehkwong1 opened the pull request:
## Description of change

Added two new fields 'Segment' and 'Sub-segment' to the companies business details section. Backend changes needed to take advantage of the addition. Works without backend changes, defaulting to show 'No segment has been added' and 'No sub-segment has been added' respectively. 

## Test instructions

_What should I see?_

## Screenshots
### Before
![image](https://user-images.githubusercontent.com/70902973/111479516-f1329e00-8728-11eb-865a-5dba190c89de.png)

### After
![image](https://user-images.githubusercontent.com/70902973/111478942-6f427500-8728-11eb-8f38-dbdb9b2e8973.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [x] Has the branch been rebased to master?
- [x] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [x] Manual compatibility testing (Browsers: Chrome, Firefox, IE11, Safari)

tomkingston reviewed (commented):
Looks good to me, but would be good if you could add `segment` and `sub_segment` to the sandbox company fixture here: https://github.com/uktrade/data-hub-frontend/blob/master/test/sandbox/fixtures/v4/company/company.json

You should probably also add a functional test to check that these results are now showing.

chloehkwong1 commented:
@tomkingston We added `segment` and `sub_segment` to `test/sandbox/fixtures/v4/company/company-one-list-corp.json` and modified the tests in `test/functional/cypress/specs/companies/business-details-spec.js` to include the two new fields.
